tests/test_loaders.py

```diff
@@ -14,6 +14,7 @@
 
 DATASETS = soundata.DATASETS
 CUSTOM_TEST_CLIPS = {
+    "warblrb10k": "759808e5-f824-401e-9058",
```

Comment:
I think here we are ordering in alphabetical order or we just don't care? haha just to be consistent :) of course it's right as it is now!